railties/lib/rails/generators/rails/plugin/plugin_generator.rb

```diff
@@ -319,10 +319,10 @@ def modules
 
       def wrap_in_modules(content)
         content = "#{content}".strip.gsub(/\W$\n/, '')
-        modules.reverse.inject(content) do |content, mod|
+        modules.reverse.inject(content) do |kontent, mod|
```

Comment:
I'll second that. `kontent` sounds like a typo (even though it mimics the `klass` convention).